services/shared/test/score/scoreCalculation.test.ts

```diff
@@ -21,4 +21,8 @@ describe('Score calculation lib test', () => {
     const roundUpNumber = roundUp(0);
     expect(roundUpNumber).toBe(0);
   });
+  it('should divide by 1, when there are no scores', () => {
```

Comment:
udpate test name, it does not check what number is used for division, it only asserts the return value, so test name should also say that.